### PR TITLE
Improve README.md

### PR DESCRIPTION
**Pull request by Panquesito7**
Title says it all.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,23 +17,23 @@ git clone --recursive https://github.com/MT-CTF/capturetheflag.git
...
+Hosting your server using the Dummy backend.
 
-### minimum
+### Minimum
 
-Hosting your server using the leveldb or redis backend
+Hosting your server using the LevelDB or Redis backend.
 
-Hosting using sqlite on an SSD or ramdisk ([with this guide](https://forum.minetest.net/viewtopic.php?f=10&t=9588))
+Hosting using SQLite on a SSD or ramdisk ([with this guide](https://forum.minetest.net/viewtopic.php?f=10&t=9588)).
```
Comment (ClobberXD) [inline README.md]: `an` is the correct choice here, as SSD phonetically starts with a vowel (`ess-ess-dee`)

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
-# CaptureTheFlag
+# Capture The Flag
 
-Uses the [CTF PvP engine](https://github.com/rubenwardy/ctf_pvp_engine)
+Uses the [CTF PvP engine](https://github.com/rubenwardy/ctf_pvp_engine).
```
Comment (ClobberXD) [inline README.md]: This line can be removed, as `ctf_pvp_engine` is now a part of this repo itself.

File: README.md (hunk 1 above)
Comment (Panquesito7) [reply README.md]: Will fix it.

File: README.md (hunk 2 above)
Comment (Panquesito7) [reply README.md]: Wouldn't be better if I just remove the link?

File: README.md [hunk 3]
```diff
@@ -17,23 +17,23 @@ git clone --recursive https://github.com/MT-CTF/capturetheflag.git
...
 
-Hosting using sqlite on an SSD or ramdisk ([with this guide](https://forum.minetest.net/viewtopic.php?f=10&t=9588))
+Hosting using SQLite on a SSD or ramdisk ([with this guide](https://forum.minetest.net/viewtopic.php?f=10&t=9588)).
 
 ## License
 
-Created by [rubenwardy](https://rubenwardy.com/)
-Code: LGPLv2.1+
+Created by [rubenwardy](https://rubenwardy.com/).
+Code: LGPLv2.1+.
```
Comment (ClobberXD) [inline README.md]: The preceding two lines aren't sentences, and as such, shouldn't have `.` at the end. It also makes the Code license confusing, as it looks like it's a decimal point after `1+`

Comment (Panquesito7) [reply README.md]: I will remove both the dots on both lines.

Comment (Panquesito7) [reply README.md]: Should I keep the dot on the first line?

File: README.md [hunk 4]
```diff
@@ -17,23 +17,23 @@ git clone --recursive https://github.com/MT-CTF/capturetheflag.git
...
 Textures: CC-BY-SA 3.0
 
 ### Mods
 
-Check out [mods/](mods/) to see all the installed mods and their respective licenses.
+Check out the [installed mods](mods/) to see them and to see their respective licenses.
```
Comment (ClobberXD) [inline README.md]: "to see them and to see their respective licences" doesn't sound right. I suggest you revert this change.

File: README.md (hunk 2 above)
Comment (Panquesito7) [reply README.md]: I kept that line, but I removed the link.

File: README.md (hunk 3 above)
Comment (Panquesito7) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (ClobberXD) [reply README.md]: Continued in https://github.com/MT-CTF/capturetheflag/pull/451#discussion_r321551596

File: README.md (hunk 3 above)
Comment (ClobberXD) [reply README.md]: I guess it can remain if you really want it to

File: docs/doc_project_overview.md [hunk 5]
```diff
@@ -3,16 +3,18 @@
...
 concepts of teams. Flags are a plugin mod, so it isn't CTF as such.
 
-# Modules in CTF_PvP_Engine
+# Modules in the CTF PvP Engine
 
 ## hudkit
 
-A support library to make the HUD API nicer.
-WTFPL.
+A support library to make the HUD API nicer.
```
[the remark this answers is not in the document]
Comment (ClobberXD) [reply docs/doc_project_overview.md]: Hm, the exact changes aren't highlighted in this page, but they're quite visible over [here](https://github.com/MT-CTF/capturetheflag/pull/451/files#diff-f6585f06932ddcdcd8bec6d13c85ddc4R10)

[the remark this answers is not in the document]
Comment (Panquesito7) [reply docs/doc_project_overview.md]: > You've (inadvertently?) added multiple trailing whitespaces to 3 lines in this file. It'd be nice if you could remove them

The `-` is the current one. The `+` is the one in this PR.
This is how it looks:

```diff
- A support library to make the HUD API nicer. License: WTFPL
+ A support library to make the HUD API nicer.
+ License: WTFPL
```
**(with markdown enabled.)**
Adding whitespaces creates a new line, even if you have already added a new line.

File: docs/doc_project_overview.md [hunk 6]
```diff
@@ -3,16 +3,18 @@
...
-A support library to make the HUD API nicer.
-WTFPL.
+A support library to make the HUD API nicer.  
+License: WTFPL.
 
 ## ctf
 
-Requires hudkit. Support for chatplus.
+Dependencies: hudkit.  
+Soft dependencies: chatplus.
```
[the remark this answers is not in the document]
Comment (Panquesito7) [reply docs/doc_project_overview.md]: I'll fix it.

File: docs/doc_project_overview.md [hunk 7]
```diff
@@ -49,22 +57,18 @@ Adds flags and flag taking.
...
 See git history.
 
-## Capture the flag
-
-more mods available in [capture the flag](http://github.com/rubenwardy/capturetheflag/).
-
-* ctf_match - adds the concept of winning, match build time,
-	and reseting the map / setting up a new game.
-	Requires ctf_flag
+## ctf_match
```
Comment (ClobberXD) [inline docs/doc_project_overview.md]: Please remove this too. A single place for mod documentation was removed from CTF because it was too outdated (see #393), and we don't really have the time to keep that section up to date. Mods in ctf_pvp_engine were documented because they rarely changed.

File: docs/doc_project_overview.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (ClobberXD) [reply docs/doc_project_overview.md]: Weird, the whitespace indeed don't exist in the actual file. It's probably being rendered wrong, sorry for the trouble.

**Comment by Panquesito7** [comment]
Fixed.

File: docs/doc_project_overview.md (hunk 7 above)
Comment (Panquesito7) [reply docs/doc_project_overview.md]: Fixed.

File: docs/doc_project_overview.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Panquesito7) [reply docs/doc_project_overview.md]: It's okay, don't worry. 🙂

**Comment by ClobberXD** [comment]
I seriously question the validity of `doc_project_overview.md`, given the recent merger:

- There's literally `ctf_pvp_engine` everywhere in that file, which doesn't exist.
- Even if all occurrences of `ctf_pvp_engine` are replaced/removed, this file now only documents a few select mods from the `ctf` modpack, which doesn't make sense.

In my opinion, this file can be completely removed, for the same reasons given in #393. Maybe @rubenwardy can express his opinions on the removal.

**Comment by Panquesito7** [comment]
> * There's literally `ctf_pvp_engine` everywhere in that file, which doesn't exist.

I also noticed that, but wasn't sure to remove it.

> In my opinion, this file can be completely removed, for the same reasons given in #393. Maybe @rubenwardy can express his opinions on the removal.

Let's wait a while until he comes, then discuss about it.
I agree to remove it, too.

BTW, thanks for the information.

**Comment by Panquesito7** [comment]
@ClobberXD, could you please add @rubenwardy as a pending reviewer?
IMHO, this would be a good idea.

**Comment by ClobberXD** [comment]
rubenwardy hasn't reviewed PRs for a very long time, and he probably never will, given that he's getting busier by the day. Also, he would've been notified of a mention in this thread, and would've replied if he did have time for reviewing CTF stuff. I'll try to pull him in though.

**Comment by Panquesito7** [comment]
> rubenwardy hasn't reviewed PRs for a very long time, and he probably never will, given that he's getting busier by the day. Also, he would've been notified of a mention in this thread, and would've replied if he did have time for reviewing CTF stuff. I'll try to pull him in though.

I see. Sorry for this (and for wasting your time).
I think it's best to wait him. Or, maybe asking the other members for their thoughts, but it's a bad idea indeed IMHO.

**Comment by Panquesito7** [comment]
@ClobberXD, what if I revert the `docs/doc_project_overview.md` changes?
I can keep the `README.md` changes only.

**Comment by ClobberXD** [comment]
That'd be great, thanks.

**Comment by Panquesito7** [comment]
> That'd be great

Done.

File: README.md [hunk 8]
```diff
@@ -17,17 +15,17 @@ git clone --recursive https://github.com/MT-CTF/capturetheflag.git
 
 ### Recommended
 
-Hosting your server using the dummy backend
+Hosting your server using the Dummy backend.
 
-### minimum
+### Minimum
 
-Hosting your server using the leveldb or redis backend
```
Comment (ClobberXD) [inline README.md]: dummy, leveldb, and redis aren't names in this context. They're the actual values to be set in `world.mt`. For this reason, I suggest reverting this change, and adding backticks instead. i.e. \`dummy\`, \`leveldb\`, etc.

Comment (ClobberXD) [inline README.md]: `dummy`, `leveldb`, and `redis` aren't names in this context. They're the actual values to be set in `world.mt`. For this reason, I suggest reverting this change, and adding backticks instead. i.e. \`dummy\`, \`leveldb\`, etc.

File: README.md [hunk 9]
```diff
@@ -17,17 +15,17 @@ git clone --recursive https://github.com/MT-CTF/capturetheflag.git
...
+Hosting your server using the LevelDB or Redis backend.
```
Comment (ClobberXD) [inline README.md]: `dummy`, `leveldb`, and `redis` aren't names in this context. They're the actual values to be set in `world.mt`. For this reason, I suggest reverting this change, and adding backticks instead. i.e. \`dummy\`, \`leveldb\`, etc.

Comment (Panquesito7) [reply README.md]: In the Minetest Wiki, they're written as-is.
Click [this link](https://wiki.minetest.net/Database_backends) to see how they're written.

Comment (ClobberXD) [reply README.md]: MT Wiki is unofficial and isn't always correct, especially when it comes to more technical details, because it's mostly maintained by the community itself and the maintainer might not be familiar with MT's technical nuances.

In this particular case, MT checks for DB backends by directly comparing the values with the lowercase names of the backends (`sqlite3`, `leveldb`, etc). If it encounters a name that doesn't match any existing names, it throws an exception (of type `BaseException`), that says `Database backend <invalid_name_here> not supported`. If you've actually tested this and can confirm that this works, great. Please only suggest stuff if you've actually tested them.

Comment (Panquesito7) [reply README.md]: I actually thought this was just a `README.md`, and users will do it like you said (`sqlite3`, etc.).
I fixed it, anyway.

Comment (ClobberXD) [reply README.md]: Thanks

File: README.md [hunk 10]
```diff
@@ -37,4 +35,4 @@ Check out [mods/](mods/) to see all the installed mods and their respective lice
 
 ## update.sh
 
-Content in this repository and its sub-modules are arranged in a manner best optimised for distribution. For all features of CTF to work properly, run update.sh instead of manually syncing the local copy. update.sh automatically pulls the latest master of the repository and all its submodules, and does some extra processing to make stuff work.
+Content in this repository and its sub-modules are arranged in a manner best optimised for distribution. For all features of CTF to work properly, run update.sh instead of manually syncing the local copy. update.sh automatically pulls the latest master of the repository and all its submodules, and does some extra processing to make stuff work.
```
Comment (ClobberXD) [inline README.md]: I'm unable to spot the difference. Has anything actually changed in this line?

Comment (Panquesito7) [reply README.md]: No, it didn't changed.
It's because there are less lines.

Comment (ClobberXD) [reply README.md]: That's weird. Git shouldn't highlight this section then. Nvm...
